notebooks/harmony subsetting/Harmony L2 Subsetter.ipynb

```diff
@@ -46,7 +46,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    if lat_max &lt;= 45 and lat_min &gt;= -45.75:
</pre><p>This should be validating against the actual limits that we got from the granule metadata.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/podaac/tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>see above, we should sue the limits set in the CMR request. This is a tutorial on how a user with a specific bounding box would use this service.</p>